synapsebridgehelpers/export_tables.py

```diff
@@ -0,0 +1,342 @@
+import synapsebridgehelpers
+import synapseclient as sc
+import numpy as np
+
+
+def replace_file_handles(syn, df, source_table_id, source_table_cols=None,
+                         content_type = "application/json"):
```

Comment:
When you call this do you first check if it is necessary to call at all?  I.e. do you try to not copy the fhs and only copy if the user is different from source table to target table?  This would be the ideal behavior.